dev/docker-compose.yaml

```diff
@@ -209,7 +209,7 @@ services:
 
 
   enrollment:
-    image: ghcr.io/uwcirg/isacc-messaging-client-sof:${ENROLLMENT_IMAGE_TAG:-latest}
+    image: ghcr.io/uwcirg/isacc-messaging-client-sof:${MESSAGING_IMAGE_TAG:-latest}
```

Comment:
We had issues when this repo used `dev` vs develop, but that should be fixed now